src/views/DrefApplicationForm/Overview/index.tsx

```diff
@@ -19,22 +21,23 @@ import Link from '#components/Link';
 import Button from '#components/Button';
 import TextInput from '#components/TextInput';
 import SelectInput from '#components/SelectInput';
-import MultiSelectInput from '#components/MultiSelectInput';
 import NumberInput from '#components/NumberInput';
 import BooleanInput from '#components/BooleanInput';
 import UserSearchMultiSelectInput, { type User } from '#components/domain/UserSearchMultiSelectInput';
 import useTranslation from '#hooks/useTranslation';
-import { useRequest, type GoApiResponse } from '#utils/restRequest';
+import { type GoApiResponse } from '#utils/restRequest';
 import {
-    stringNameSelector,
-    numericIdSelector,
     stringValueSelector,
 } from '#utils/selectors';
 import useGlobalEnums from '#hooks/domain/useGlobalEnums';
 import useCountry from '#hooks/domain/useCountry';
 import NationalSocietySelectInput from '#components/domain/NationalSocietySelectInput';
 import CountrySelectInput from '#components/domain/CountrySelectInput';
 import DisasterTypeSelectInput from '#components/domain/DisasterTypeSelectInput';
+import DistrictSearchMultiSelectInput,
+{
+    type DistrictItem,
+} from '#components/domain/DistrictSearchMultiSelectInput';
```

Comment:
Let's avoid using prettier on file level
```suggestion
import DistrictSearchMultiSelectInput, {
    type DistrictItem,
} from '#components/domain/DistrictSearchMultiSelectInput';
```